source/includes/connect/stable-api.php

```diff
@@ -0,0 +1,23 @@
+// start-specify-v1
+<?php
+
+$uri = "mongodb://<hostname>:<port>";
+
+$driverOptions = [
+    'serverApi' => new MongoDB\Driver\ServerApi('1')
```

Comment:
```suggestion
    'serverApi' => new MongoDB\Driver\ServerApi('1'),
```

If you're breaking PHP arrays into multiple lines, it's consistent to include a trailing comma after each element. Alternatively, you can inline the entire array on a single line:

```php
$driverOptions = ['serverApi' => new MongoDB\Driver\ServerApi('1')];
```